Adds vars for XBian

Gordin opened the pull request:
I just copied the Debian.yml to XBian.yml to make this work on an XBian installation.

XBian is a debian/raspbian based distribution for XBMC and it reports itself as this:
PRETTY_NAME="XBian 1.0 (knockout)"
NAME=XBian
VERSION_ID=1.0
VERSION="1.0 (knockout)"
ID=raspbian
ID_LIKE=debian
ANSI_COLOR="1;31"
HOME_URL="http://www.xbian.org/"
SUPPORT_URL="http://forum.xbian.org/"
BUG_REPORT_URL="https://github.com/xbianonpi/xbian/issues"

stefangweichinger commented:
So `{{ ansible_os_family }}` is `XBian` for that host?
pls check with some playbook returning these facts.
In fact I'd like to know the values of:

- {{ ansible_distribution }}
- {{ ansible_distribution_major_version }}
- {{ ansible_os_family }}

I don't think there is a docker image for that OS to let molecule test the role against?

Gordin commented:
Honestly I don't know enough about ansible to even check the values of these variables... I just got an error about ansible not finding variables, the vars directory contained yml files named after distributions, so I looked up what the XBian system calls itself, renamed Debian.yml to XBian.yml and then it worked... It doesn't look like there is a docker image for XBian to test this.

I guess you can close the PR then, I've already moved away from XBian (because it's using upstart, nothing to do with ansible), so I won't put any more time in this.

stefangweichinger commented:
Ok, I see. While it would be nice to support even more flavours of linux, it has to be maintained and tested as well. So yes: thanks for the PR, but I close this unmerged for now.
